src/styles.css

```diff
@@ -1,9 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
 
-body {
-    font-size: 0.9rem;
-    font-family: Arial, serif;
-}
+@import "./sass/styles.scss";
```

Comment:
can't we have just a styles.scss and use that? or is angular.json is limited to use css?

Reply:
the program crashed when I tried to make it styles.scss, I couldn't figure out where it was crashing though - can take another look